WormBrowser/OWSearchViewController.m

```diff
@@ -82,7 +82,7 @@ - (void)viewDidLoad
 
 -(void) viewDidAppear:(BOOL)animated
 {
-    [TestFlight passCheckpoint:@"Search view displayed"];
+
```

Comment:
if deleting the TestFlight line, might as well delete the method, particularly b/c it's not even calling super`.

Might as well delete the unused `didReceiveMemoryWarning`